go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/st8ed/aws-cost-exporter
 
-go 1.17
+go 1.18
```

Comment:
bump the minimum go version to enable using the `atomic` package with generics

Reply:
The `sync.atomic.Pointer` [was added in Go 1.19](https://pkg.go.dev/sync/atomic@go1.19#Pointer), need to bump this to `1.19`.